dictdiffer/__init__.py

```diff
@@ -9,23 +9,33 @@
 
 """Dictdiffer is a helper module to diff and patch dictionaries."""
 
-import sys
 import copy
 
-if sys.version_info[0] == 3:  # pragma: no cover (Python 2/3 specific code)
-    string_types = str,
-else:  # pragma: no cover (Python 2/3 specific code)
-    string_types = basestring,
+from utils import (WildcardDict,
```

Comment:
I would be nice to use relative imports everywhere (`.utils`, `.wrappers`, etc.) and sort imports according to PEP8.